functions/__fzf_search_history.fish

```diff
@@ -4,7 +4,9 @@ function __fzf_search_history --description "Search command history. Replace the
     set command_with_ts (
         # Reference https://devhints.io/strftime to understand strftime format symbols
         builtin history --null --show-time="%m-%d %H:%M:%S | " |
-        fzf --read0 --tiebreak=index --query=(commandline) |
+        fzf --read0 --tiebreak=index --query=(commandline) \
+            --preview="echo {4..}" \
```

Comment:
How does `echo {4..}` result in the just the command being displayed? Wondering if there's a fish syntactical trick I can do to `set command_selected (string split --max 1 " | " $command_with_ts)[2]` below

Reply:
`{4..}` is an fzf trick not related to fish. fzf replace `{}` in the preview by the selected line. You can also index the line by field (by default it's split on whitespace, this can be changed with `--delimiter`). `{4..}` means from the fourth field to the end of the selected line